versions.props

```diff
@@ -3,7 +3,7 @@ com.github.ben-manes.caffeine:caffeine = 3.1.8
 com.google.auto.service:auto-service = 1.1.1
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.errorprone:error_prone_annotations = 2.11.0
-com.google.guava:guava = 32.1.2-jre
+com.google.guava:guava = 32.1.3-jre
```

Comment:
Guava 32.1.1 breaks the PJF 2.38.0 integration in Spotless due to an issue about resolving transitive dependencies, which has been fixed in Guava 32.1.3. It would be nice to have a revision release for this update. 

See:
- https://github.com/google/guava/issues/6657.
- https://github.com/google/guava/releases/tag/v32.1.3.